plugins/streaming/impls/jms/src/main/java/org/pentaho/di/trans/step/jms/ui/CheckBoxTableComboDefaultButton.java

```diff
@@ -80,8 +90,10 @@ public CheckBoxTableComboDefaultButton( Composite parentComposite, PropsUI props
 
     wUseDefaultCheckBox.addSelectionListener( new SelectionListener() {
       @Override public void widgetSelected( SelectionEvent selectionEvent ) {
+        //boolean selection = ( (Button) selectionEvent.getSource() ).getSelection();
```

Comment:
probably want to remove this commented line

Reply:
Done!